site/content/contribute/webapp/developer-setup.md

```diff
@@ -5,24 +5,35 @@ weight: 2
 subsection: Web App
 ---
 
-# Developer Setup
+Set up your development environment for building, running, and testing the Mattermost web app.
 
-## Prerequisites
-- Git
-- homebrew (for Mac)
+#### Prerequisites
 
-## Setup
+1. Set up your [development environment for the Mattermost server](/contribute/server/developer-setup). 
 
-Follow these steps to set up your web app developer environment:
+ - Running these steps will also install Git on your machine, and Homebrew for Mac.
 
-1. To develop for the web app you're also going to need to [set up the developer environment for the server](/contribute/server/developer-setup). So if you haven't done that yet, start there and then come back here.
+2. Install Node.js and npm following the instructions at https://www.npmjs.com/get-npm.
```

Comment:
I suggest specifying the min required version. The nodejs website actually recommends downloading 8.12.0, but that fails the unit tests. Perhaps:
```2. Install Node.js and npm following the instructions at https://www.npmjs.com/get-npm, using node version 10.x or later.```